### PR TITLE
Changed  d in "Firefox differentiators" to Firefox Differentiators

### PR DESCRIPTION
**Pull request by Rish-av4**
> Issue number that this PR fixes (if any). For example: 'Fixes #987654321'

> What was wrong/why is this fix needed? (quick summary only)

> Anything else that could help us review it

### CODE REVIEW

**Comment by Rish-av4** [comment]
I thought it would look nice for the UX pov . It's a hefty term and keeping it lowercase is no justice. I am new to open source and currently trying my hands on small typos and am learning.

**Comment by Rish-av4** [comment]
I have also added one word in a sentence that now makes sense.

File: files/en-us/mozilla/add-ons/webextensions/firefox_differentiators/index.html [hunk 1]
```diff
@@ -38,7 +38,7 @@
   </tr>
   <tr>
    <td>
-    <p>Help users locate content of interest in a web page, with the ability to <a href="/en-US/docs/Mozilla/Add-ons/WebExtensions/API/find">find and highlight text in a webpage</a>.</p>
+    <p>Help users locate content of their interest in a web page, with the ability to <a href="/en-US/docs/Mozilla/Add-ons/WebExtensions/API/find">find and highlight text in a webpage</a>.</p>
```
Comment (hamishwillee) [inline files/en-us/mozilla/add-ons/webextensions/firefox_differentiators/index.html:41]: While both versions are correct English, the original reads better.

File: files/en-us/mozilla/add-ons/webextensions/firefox_differentiators/index.html [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Firefox differentiators
+title: Firefox Differentiators
```
Comment (hamishwillee) [inline files/en-us/mozilla/add-ons/webextensions/firefox_differentiators/index.html:2]: We use sentence style capitalisation for headings: https://developer.mozilla.org/en-US/docs/MDN/Guidelines/Writing_style_guide#title_and_heading_capitalization

**Comment by hamishwillee** [comment]
Thanks for contributing. I'm rejecting these fixes as not consistent with style guides.

**Pull request closed without merging**